lib/backpex/live_components/form_component.ex

```diff
@@ -309,7 +313,7 @@ defmodule Backpex.FormComponent do
     end
   end
 
-  defp handle_save(socket, :resource_action, params) do
+  defp handle_save(socket, :resource_action, params, _) do
```

Comment:
Please add parameter names.

btw... There should also be a Credo rule for that. Maybe you can enable it (if its not to much work in the rest of the codebase)?